runtime/compiler/control/CompilationThread.cpp

```diff
@@ -6918,10 +6940,15 @@ TR::CompilationInfoPerThreadBase::preCompilationTasks(J9VMThread * vmThread,
    // Note that currently we don't support out-of-process relocatable compilations
    if (canDoRelocatableCompile)
       {
-      _vm = TR_J9VMBase::get(_jitConfig, vmThread, TR_J9VMBase::AOT_VM);
       entry->_useAotCompilation = true;
 
-      if (_compInfo.getPersistentInfo()->getJITaaSMode() == CLIENT_MODE && enableJITaaSRemoteAOT)
+      if (_compInfo.getPersistentInfo()->getJITaaSMode() == SERVER_MODE)
```

Comment:
The comment `// Note that currently we don't support out-of-process relocatable compilations` needs to be deleted

Reply:
Should testing `_compInfo.getPersistentInfo()->getJITaaSMode() == SERVER_MODE` be changed to `entry->isOutOfProcessCompReq()` here as well?

Reply:
yes, since we have a valid entry we can issue the test based on entry